### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by lilGuzi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,3 @@
...
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+3750c7a03a8b0a175a00ab24263df356de54043ebf51d9cebcf05db974b9ec0c5a13546abc3cac9856f59b521ec73c4b2ae003d9414040f1c98c859eec73ab15
```
Comment (elysemontano) [inline README.md]: Is this needed?

**Review by elysemontano — CHANGES_REQUESTED** [review]
I am seeing a lot of work that goes beyond what the Trello card is calling for.  For instance, I am seeing some JWT configs and your controller methods.  This PR should be ONLY validations and model specs

File: README.md (hunk 1 above)
Comment (lilGuzi) [reply README.md]: updated Trello cards to include controller methods. also removed the seceret key from the README.

File: spec/models/basketball_court_spec.rb [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+    email: 'test@example.com',
+    password: 'password',
+    password_confirmation: 'password',
+    skill_level: 1
+    )
+  }
+  it 'should validate street' do
+    attributes = [:name, :address, :city, :state, :zip, :court_type, :number_players]
+    basketball_court = user.basketball_courts.create()
+    attributes.each do |attr|
```
Comment (elysemontano) [inline spec/models/basketball_court_spec.rb:14]: Cool approach!

**Review by elysemontano — APPROVED** [review]

**Pull request merged**